Dockerfile.heroku

```diff
@@ -6,13 +6,14 @@ RUN apt-get update && \
     && rm -rf /var/lib/apt/lists/* && \
     gem install bundler --no-document && \
     bundle config set no-cache 'true' && \
-    bundle config set silence_root_warning 'true'
+    bundle config set silence_root_warning 'true' && \
+    bundle config set --local deployment 'true'
```

Comment:
remote:  ---> Running in b16da39cfad6
remote: [DEPRECATED] The `--deployment` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set --local deployment 'true'`, and stop using this flag